launcher/ui/pages/instance/ExternalResourcesPage.cpp

```diff
@@ -0,0 +1,297 @@
+#include "ExternalResourcesPage.h"
+#include "ui_ExternalResourcesPage.h"
+
+#include "DesktopServices.h"
+#include "Version.h"
+#include "minecraft/mod/ModFolderModel.h"
+#include "ui/GuiUtil.h"
+
+#include <QKeyEvent>
+#include <QMenu>
+
+namespace {
+// FIXME: wasteful
+void RemoveThePrefix(QString& string)
+{
+    QRegularExpression regex(QStringLiteral("^(([Tt][Hh][eE])|([Tt][eE][Hh])) +"));
```

Comment:
Case insensitive regex exists, don't spam capture groups, prefer non-capturing groups
```suggestion
    QRegularExpression regex(QStringLiteral("^(?:the|teh) +"),QRegularExpression::CaseInsensitiveOption);
```

Reply:
I wonder why it removes "teh" as well. What mod is called "Teh <something>"

Reply:
![image](https://user-images.githubusercontent.com/9145768/176645922-ec7f9764-47e0-4078-b45a-bf742c934b5d.png)
the commit and the issues only talk about 'The', so idk :/